composer.json

```diff
@@ -11,10 +11,11 @@
     ],
     "require" : {
         "php": ">=5.4",
+        "swiftmailer/swiftmailer": "^5.1",
         "tijsverkoyen/css-to-inline-styles": "^1.5"
     },
     "require-dev" : {
-        "swiftmailer/swiftmailer": "^5.1"
+        "phpunit/phpunit": "^4.0"
```

Comment:
@hkdobrev 

EDIT :  phpunit 5.x requires php >=5.6. So if you want to be compatible with php >=5.4 you need to stay with phpunit 4.x or allow both versions.